src/RPackage-Core/CompiledCode.extension.st

```diff
@@ -1,23 +1,27 @@
 Extension { #name : #CompiledCode }
 
 { #category : #'*RPackage-Core' }
-CompiledCode >> package [
-	^ self packageFromOrganizer: RPackage organizer
+CompiledCode >> extensionPackage [
```

Comment:
One of the next step is to merge RPackage into Kernel when it will be clean and rid of the SystemAnnouncer. When it'll be done I wanted to put the method in a private protocol